pages/contact.html

```diff
@@ -57,5 +59,55 @@ <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
     <p>School Location on Google maps</p>
     <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
 </div>
+
+<div class="ui vertical footer segment">
```

Comment:
I would suggest maybe looking at adding some `padding-top` to this segment, as it is very close to body page